### DISCUSSION TITLE
Feedback Portal not working (just grey)

### DISCUSSION THREAD

**Discussion by KvN069** [discord-tickets/bot#449] [Support (Q&A)] [answered]
Hello, I have setup the bot today and everything works fine so far. The only problem I am facing right now is, that if I click on the feedback portal it will redirect me to a page which is just showing grey without anything in it. Also the titel of the page is "SERVER NAME HERE". Anyone know how I can fix that?

![Bug](https://github.com/discord-tickets/bot/assets/20592607/898fb1c5-b301-4a9b-828d-4ae822f3e859)

**Comment by eartharoid** [comment, marked as answer]
@DerBoss069 the feedback page hasn't been made yet. Only `/settings` is currently functional.
Feedback is being stored though so when the update comes you will be able to see all of the feedback submitted before.

**Reply by KvN069** [reply]
Sounds awesome, thanks for your work and for the quick reply!
